src/site/markdown/getting-started.md

```diff
@@ -17,58 +17,66 @@
 
 ## Getting Started
 
-To on-board incremental Maven you need to complete several steps:
+To onboard incremental Maven you need to complete several steps:
```

Comment:
`incremental Maven` -> `the build-cache extension`

Reply:
fixed